src/components/Fields/TextArea.test.js

```diff
@@ -100,9 +100,14 @@ describe('TextArea', () => {
         expect(textArea).to.have.className('uir-text-area--full-width');
     });
 
-    it('has a auto resize class if hasAutoHeight is passed', () => {
-        const textArea = shallow(<TextArea hasAutoHeight />);
-        expect(textArea).to.have.className('uir-text-area--has-auto-height');
+    it('has a auto resize class if hasFixedHeight is true', () => {
+        const textArea = shallow(<TextArea hasFixedHeight />);
+        expect(textArea).to.have.className('uir-text-area--has-fixed-height');
+    });
+
+    it('does not have a auto resize class if hasFixedHeight is false', () => {
+        const textArea = shallow(<TextArea hasFixedHeight={false} />);
+        expect(textArea).to.not.have.className('uir-text-area--has-fixed-height');
```

Comment:
Both of these tests can now be removed since you no longer use the `--had-fixed-height` class.
It would be nice to have another test because effectively we're no longer testing the `hasFixedHeight` prop. But I'm not sure if it's possible to test if `autosize()` has been called or not.